README.md

```diff
@@ -40,20 +39,33 @@ After composer updates its autoload file, and that happens after `dump-autoload`
 Composer plugin:
 
 - Scans installed packages for `config-plugin` extra option in their `composer.json`.
-- Copies missing config files into the project `configs`.
-- Writes a merge plan into `config/packages/merge_plan.php`. It includes configuration from each package `composer.json`.
-- Tracks change to configuration files from the vendor by storing metadata in the `config/packages/dist.lock` file.
-- In the interactive console mode it asks what to do with modified configs after updating packages in the vendor.
+- Writes a merge plan into `config/MERGE_PLAN.php`. It includes configuration from each package `composer.json`.
   
 In the application entry point, usually `index.php`, we create an instance of config loader and require a configuration
 we need:
 
 ```php
-$config = new \Yiisoft\Config\Config(dirname(__DIR__));
+use Yiisoft\Config\Config;
+use Yiisoft\Config\ConfigPaths;
+
+$config = new Config(
+    new ConfigPaths(dirname(__DIR__)),
```

Comment:
Usually it's a single root path so we can revert it to previous signature that doesn't require `new ConfigPaths()`.